tests/unit/amo/components/TestCollectionManager.js

```diff
@@ -268,6 +268,18 @@ describe(__filename, () => {
       .toHaveProp('disabled', true);
   });
 
+  it('disables submit button when the slug is blank', () => {
+    const root = render();
+
+    // Enter in a blank collection name.
+    typeInput({ root, name: 'slug', text: '' });
```

Comment:
I'd like to see a test for a slug which isn't empty, but contains only spaces. We should add one for the name field too, if we don't have one already. I think it's fine to just add it to this case.